rust/cardano-blockchain-types/src/metadata/cip36/mod.rs

```diff
@@ -227,71 +227,85 @@ impl Cip36 {
 
     /// Get the `is_cip36` flag from the registration.
     /// True if it is CIP-36 format, false if CIP-15 format.
+    #[must_use]
```

Comment:
I have no idea why Clippy didn't complain before and fails now.

Reply:
Not sure too 😅